homework/java/zhdanok_liudmila/lecture2/task1/src/main/java/by/jrr/mytests/servise/DogServise.java

```diff
@@ -0,0 +1,24 @@
+package by.jrr.mytests.servise;
+
+import by.jrr.mytests.bean.Dog;
+
+public class DogServise {
+
+    Dog dog;
+
+    public DogServise(Dog dog) {
+        this.dog = dog;
+    }
+
+    public String bark () {
+        return dog.getName() + " BARK!";
```

Comment:
Usually we do not return calculation. For string it's better to prepare it in advance.
For String concatenations use method "".concat("BARK");

Best practices in return to use String.format("Dog %s says %s!", ownerName, dogVoice)